v0/destinations/indicative/transform.js

```diff
@@ -0,0 +1,63 @@
+const { EventType } = require("../../../constants");
+const { CONFIG_CATEGORIES, MAPPING_CONFIG } = require("./config");
+const {
+  defaultRequestConfig,
+  getFieldValueFromMessage,
+  constructPayload,
+  defaultPostRequestConfig
+} = require("../../util");
+
+const responseBuilderSimple = (message, category, destination) => {
+  const payload = constructPayload(message, MAPPING_CONFIG[category.name]);
+  const responseBody = { ...payload, apiKey: destination.Config.apiKey };
+  const response = defaultRequestConfig();
+  response.endpoint = category.endPoint;
+  response.method = defaultPostRequestConfig.requestMethod;
+  response.headers = {
+    "Content-Type": "application/json"
+  };
+  response.userId = getFieldValueFromMessage(message, "userId");
```

Comment:
this I feel should always be anonymousId, that way may help in server side to not switch between rt job workers, wdyt

Reply:
Agreed, but that way, we have to be consistent across destinations. Let's consider this as a part of refactoring.

Reply:
yes..let's take a note